lib/ansible/modules/cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -58,9 +60,9 @@
         If 'gateway_id' is specified, you can refer to the VPC's IGW by using the value 'igw'.
         Routes are required for present states.
     type: list
-    elements: str
+    elements: dict
```

Comment:
Woopsie. Did we have an ignore.txt entry for this one?

Reply:
It's not defined in the argument_spec at all.  We currently have a whopping 3 instances of elements being defined in the argument_spec.  I've started adding it to the docs as I've been cleaning up because @felixfontein asked me to in one of the reviews.

Reply:
Ah. Thanks for getting rid of the ignore.txt entries.